lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -3745,6 +3745,7 @@ JITManager::HandleServerCallResult(HRESULT hr, RemoteCallType callType)
         return true;
     case E_ABORT:
         throw Js::OperationAbortedException();
+    case HRESULT_FROM_WIN32(ERROR_COMMITMENT_LIMIT): 
```

Comment:
that's cool, I didn't know you could use constexpr functions in case statements.